solutions/bio2/BrokersDoom/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { Link, SCode, SCodeBlock, SText, STitle } from "components";
+import { brace } from "solutions/util";
+
+export const BrokersDoom = () => {
+  return (
+    <>
+      <STitle>Abridged Statement</STitle>
+      <SText>
+        You are given an array of numbers, and an integer $t$. 
+        Find the minimum achievable profit that is greater 
+        than or equal to $t$, where profit can be earned
+        by buying an item, waiting for some number of day 
+        and then selling it, or by selling an item and then buying it.
+      </SText>
+
+      <STitle>Observation</STitle>
+      <SText>
+        This problem is equivalent to creating a {<Link href="https://usaco.guide/silver/prefix-sums?lang=cpp">prefix sum</Link>} array
+        and finding the minimum difference between two elements 
+        in the array that is greater than or equal to $t$. This 
+        is because the profit is the absolute value of the difference
+        between the value of the item at two points in time (since a
+        negative difference indicates a loss, and a loss is equivalent 
+        to the same profit, but instead of buying the item then selling
+        it, you can sell it and then buy it), and the value of the 
+        item at a point in time is the prefix sum of the changes in 
+        value up to that point in time.
+      </SText>
+
+      <STitle>Sample Explanation</STitle>
+      <SText>
+        In the input, the array is $[-5, 9, 9, -6, -4, -5, 100]$ and $t = 19$.
+      </SText>
+      <SText>
+        Therefore, the prefix sum array is $[0, -5, 4, 13, 7, 3, -2, 98]$.
+      </SText>
+      <SText>
+        The minimum difference between two elements in the array that is greater than or equal to $19$ is $98 - 13 = 85$.
+      </SText>
+      <STitle>Solution</STitle>
+      <SText>
+        Of course, simply iterating through all pairs of elements in 
+        the prefix sum array and checking if the difference is 
+        greater than or equal to $t$ will work, but this is 
+        too slow for any non-trivial subtasks. Instead, we can sort the 
```

Comment:
Please remove "for any non-trivial subtasks" because there aren't any subtasks mentioned in this question. Just writing "too slow" or "too slow given the constraint on `$n$`" should be good.

Reply:
there aren't, but there is "easy, medium hard" on the BIO2 grader.... what do you think about that...

Reply:
Sure, but we have no idea what the constraints on $n$ are in the different subtasks, and it's mentioned nowhere in the problem statement, so I think it's better not to mention subtasks. It's not like we're solving separately for different subtasks here like on a classic IOI-style problem.

Reply:
ah, the beauty of BIO2 😭